release.sh

```diff
@@ -74,10 +74,6 @@ if ! command git diff --stat --exit-code HEAD; then
     echo "Please make sure you have no uncommitted changes" >&2
     : $((++INSANE))
 fi
-if ! command npm owner ls katex | grep -q "^$(command npm whoami) <"; then
-    echo "You don't seem do be logged into npm, use \`npm login\`" >&2
-    : $((++INSANE))
-fi
```

Comment:
Is there no equivalent `yarn` command for this?

Reply:
There is `yarn login`: https://yarnpkg.com/lang/en/docs/publishing-a-package/
I'm not sure about `whoami` but it ought to be possible.  `yarn info` might also be a possible replacement for `npm owner ls`...

Reply:
@edemaine Yarn's login is different from npm's login. It's more like `git config --global user.name "John Doe"; git config --global user.email johndoe@example.com`. 

> Later when you run a command that requires authentication such as `yarn publish`, you will have to enter your password to do so.

Reply:
I found that `yarn config get username` works like `npm whoami` and `yarn info katex maintainers` is pretty close to `npm owner ls katex`.  Unfortunately, `yarn info katex maintainers` is out of date (missing me in particular), though the same was true of `npm owner ls` until I bugged NPM.

Reply:
@edemaine Have you tried  `yarn owner list katex`?

Reply:
Oh, nice!  That list is up-to-date for me.

Reply:
@edemaine AFAIK, the purpose of this check is that one can't `npm publish` without `npm login`. However, it is possible to `yarn publish` without `yarn login`.